commit 1999821fa5008a95a16404ee35fb08ec451a8180
Author: Sabrina Dubroca <sd@queasysnail.net>
Date:   Tue Aug 22 15:36:08 2017 +0200

    macsec: add genl family module alias
    
    
    [ Upstream commit 78362998f58c7c271e2719dcd0aaced435c801f9 ]
    
    This helps tools such as wpa_supplicant can start even if the macsec
    module isn't loaded yet.
    
    Fixes: c09440f7dcb3 ("macsec: introduce IEEE 802.1AE driver")
    Signed-off-by: Sabrina Dubroca <sd@queasysnail.net>
    Signed-off-by: David S. Miller <davem@davemloft.net>
    Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>

diff --git a/drivers/net/macsec.c b/drivers/net/macsec.c
index 79411675f0e6..d16ce61b3696 100644
--- a/drivers/net/macsec.c
+++ b/drivers/net/macsec.c
@@ -3518,6 +3518,7 @@ module_init(macsec_init);
 module_exit(macsec_exit);
 
 MODULE_ALIAS_RTNL_LINK("macsec");
+MODULE_ALIAS_GENL_FAMILY("macsec");
 
 MODULE_DESCRIPTION("MACsec IEEE 802.1AE");
 MODULE_LICENSE("GPL v2");
diff -uN a/1.txt b/1.txt
--- a/dummy/rpi_1003_1999821fa5008a95a16404ee35fb08ec451a8180.txt	1970-01-01 00:00:00.000000000 +0000
+++ b/dummy/rpi_1003_1999821fa5008a95a16404ee35fb08ec451a8180.txt	2013-12-23 04:07:40.000000000 +0000
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
